### ISSUE TITLE
Video conversion issue with rights on user folder

### ISSUE THREAD

**Issue by albertoSoto** [observesport/lince-plus#41]
A video conversion error happens, but a confirmation is shown.
We should:
- we should inform the user beforehand
- check user perm before starting conversion
- show confirmation message if the file has been able to be saved, now it happens always

![image](https://github.com/observesport/lince-plus/assets/3663174/bfbecbc4-bd7e-412a-962c-47d71f5b548c)

**Comment by albertoSoto** [comment]
This feature is not a problem. It won't fix, user need to rely on the perms given by the OS to the application
